decidim-meetings/spec/system/meeting_questions_administrate_spec.rb

```diff
@@ -142,7 +144,8 @@ def questionnaire_edit_path
   end
 
   def open_first_question
-    expect(page).to have_css(".meeting-polls__question--admin", match: :first)
+    expect(page).to have_css(".meeting-polls__question--admin:not([disabled])", visible: :visible)
+    sleep(2)
```

Comment:
I see that we still have the `sleep` call here. I understand that we can't get rid of it, right?

Reply:
Yes, the problem is that without the sleep the test clicks on the element at the same moment a request is sent to refresh and at this moment the question can't be open as [commented](https://github.com/decidim/decidim/pull/13028#issuecomment-2180970083)

Reply:
Oh I missed that comment, thanks